### ISSUE TITLE
Child Theme Setup Documentation

### ISSUE THREAD

**Issue by VivekShingala** [SnowdogApps/magento2-alpaca-theme#441]
Hi Everyone

I am new to theme development in Magento. By following the README file, many things are not clear.

Is there any better/clear documentation to set up the child theme? It would be of great help.

**Comment by ggm-dev** [comment]
I totally agree - Here's hoping that the README and documentation can get some clarification.  Or perhaps there's a community member out there who has successfully created a child theme and would like to chime in . . . I found some issues here on GitHub that mentioned a bootstrap template to help with this, but I can't find any evidence that it exists yet.  Fingers crossed - great looking theme and very generous of @snowdog and the snowdog folks to release it as open source.

**Comment by ggm-dev** [comment]
Hi all - any updates on this front?

**Comment by anqaka** [comment]
We are working on boilerplate tools and full guide for building child theme. It should be ready next week. Sorry for missing updates from our side.

**Comment by anqaka** [comment]
Hi, we have published tools to setup child theme based on Alpaca theme:
https://www.npmjs.com/package/@snowdog/create-alpaca-theme
In case of any problems, please create an issue: https://github.com/SnowdogApps/create-alpaca-theme
We are working on refactoring and updating docs, it will be merge soon. Hope it helps. 
Any feedback will be helpful.

**Comment by ofrankowska** [comment]
Hi, new docs just went live https://magento2-alpaca-docs.vercel.app/. Feedback would be much appreciated :)
